mail.pohjoistech.fi.email.json

```diff
@@ -0,0 +1,56 @@
+{
+  "providerId": "mail.pohjoistech.fi",
+  "providerName": "PohjoisMail",
+  "serviceId": "email",
+  "serviceName": "Mail",
+  "version": 1,
+  "syncPubKeyDomain": "domainconnect.pohjoistech.fi",
+  "syncRedirectDomain": "mail.pohjoistech.fi",
+  "logoUrl": "https://r2.pohjoistech.fi/logo.png",
+  "description": "Enables a domain to work with PohjoisMail, a mail service offered by Pohjoistech.",
+  "variableDescription": "ip is the IP address of the server, and key is the DKIM public key.",
+  "records": [
+    {
+      "pointsTo": "%ip%",
+      "type": "A",
+      "host": "pohjoismail",
+      "essential": "OnApply",
+      "ttl": 14400
+    },
+    {
+      "pointsTo": "autodiscover",
+      "type": "CNAME",
+      "host": "pohjoismail",
+      "essential": "OnApply",
+      "ttl": 14400
+    },
+    {
+      "pointsTo": "mailsrv.pohjoistech.fi",
+      "type": "MX",
+      "host": "@",
+      "essential": "OnApply",
+      "ttl": 14400,
+      "priority": 10
+    },
+    {
+      "spfRules": "a mx ip4:193.108.130.22 include:relay.mailchannels.net ip6:2605:6400:3:fed5:1000:201:0:2",
+      "type": "SPFM",
+      "host": "@",
+      "essential": "OnApply"
+    },
+    {
+      "type": "TXT",
+      "host": "@",
+      "data": "v=DMARC1; p=quarantine; rua=mailto:dmarc@%domain%; ruf=mailto:dmarc-forensics@%domain%; sp=reject; fo=1; adkim=s; aspf=s",
```

Comment:
For DMARC I would recommend txtConflictMatchingMode set to  All to avoid double entry